Kickstarter-iOS/Views/Cells/SettingsFollowCell.swift

```diff
@@ -35,7 +35,8 @@ internal final class SettingsFollowCell: UITableViewCell, ValueCell {
     }
 
     _ = self.separatorView
-      |> separatorStyle
+      ||> UIView.lens.backgroundColor .~ .ksr_grey_500
```

Comment:
Since we are using this configuration over and over through the settings cells, I would create a property `settingsSeparatorStyle` on the `SettingsStyles.swift` file:
```
public let settingsSeparatorStyle = UIView.lens.backgroundColor .~ .ksr_grey_500
  <> UIView.lens.accessibilityElementsHidden .~ true
```

And reuse it like:
```
_ = self.separatorView
      ||> settingsSeparatorStyle
```